_specs/ecip-1109.md

```diff
@@ -2,8 +2,8 @@
 lang: en
 ecip: 1109
 title: Spiral EVM and Protocol Upgrades
-status: Last Call
-type: Meta
+status: Accepted
+type: Core
```

Comment:
`_specs/ecip-1109.md is NOT valid:	 {:type=>["is not included in the list"]}`

https://github.com/ethereumclassic/ECIPs/actions/runs/7481139299/job/20362096853?pr=524#step:4:259

https://github.com/ethereumclassic/ECIPs/blob/master/_specs/ecip-1000.md#ecip-types

Reply:
``ruby
    validates_inclusion_of :category,
                           in: %w(Core Networking Interface ERC ECBP),
                           if: Proc.new { |v| v.type == 'Standards Track' }
    validates_inclusion_of :type, in: ['Standards Track', 'Informational', 'Meta']
```

https://github.com/meowsbits/ecip_validator/blob/master/lib/ecip_validator/validator.rb#L24-L27

Reply:
reverted back to Meta